site/content/docs/main/api-types/schedule.md

```diff
@@ -78,6 +78,11 @@ spec:
     # a default value of 30 days will be used. The default can be configured on the velero server
     # by passing the flag --default-backup-ttl.
     ttl: 24h0m0s
+    # The labels you want on backup objects, created from this schedule (instead of copying the labels you have on schedule object itself).
+    # Setting this will also ensure your backup object does not get labels from f.ex. argocd copied to your backup object (making argocd think it created it and wanting to remove it)
```

Comment:
How about this instead of the second sentence?
# Setting this will also ensure your backup object does not get labels from f.ex. argocd copied to your backup object (making argocd think it created it and wanting to remove it) ->
# When this field is set, the labels from the Schedule resource are not copied to the Sackup resource.

Reply:
updated.